searx/engines/torznab.py

```diff
@@ -8,56 +8,64 @@
 
 """
 
+from typing import List, Dict, Optional, Any
 from datetime import datetime
 from urllib.parse import quote
 from lxml import etree
+import requests
 
 from searx.exceptions import SearxEngineAPIException
 
-# about
-about = {
+# engine settings
+about: Dict[str, Any] = {
     "website": None,
     "wikidata_id": None,
     "official_api_documentation": "https://torznab.github.io/spec-1.3-draft",
     "use_official_api": True,
     "require_api_key": False,
-    "results": 'XML',
+    "results": "XML",
```

Comment:
Please do not change the quotes if there is not a need for.

Reply:
Done in [02bf446](https://github.com/searxng/searxng/pull/2514/commits/02bf4469c0db0b0d856b88d7a632646d147d275d)